onnx_coreml/_operators_nd.py

```diff
@@ -16,7 +16,10 @@
 from ._operators import _convert_abs, _convert_relu, _convert_sqrt, _convert_exp, \
                         _convert_elu, _convert_selu, _convert_sigmoid, _convert_sign, \
                         _convert_prelu, _convert_conv, _convert_bn, _convert_upsample, \
-                        _convert_pool
+                        _convert_pool, _convert_softsign, _convert_softplus, _convert_log, \
```

Comment:
all the other ops are okay, but for `pool` we would need to add a new function. Because the existing op is dependent on the rank 5 assumption, in the way it handles 1D and 2D pooling.